crates/cargo-test-support/src/publish.rs

```diff
@@ -155,3 +156,90 @@ pub fn validate_crate_contents(
         }
     }
 }
+
+pub(crate) fn create_index_line(
+    name: serde_json::Value,
+    vers: &str,
+    deps: Vec<serde_json::Value>,
+    cksum: &str,
+    features: crate::registry::FeatureMap,
+    yanked: bool,
+    links: Option<String>,
+    v: Option<u32>,
+) -> String {
+    // This emulates what crates.io may do in the future.
```

Comment:
This comment is a little out of date. crates.io now performs this transformation. 

```suggestion
    // This emulates what crates.io does to retain backwards compatibility.
```